lib/modelConfirmationGUI.py

```diff
@@ -54,5 +59,12 @@ def paste_model_confirmation_gui():
             return
         else:
             actions.user.paste(GPTState.text_to_confirm)
+            GPTState.last_response = GPTState.text_to_confirm
+            GPTState.last_was_pasted = True
             GPTState.text_to_confirm = ""
             confirmation_gui.hide()
+
+    def chain_model_confirmation_gui():
```

Comment:
If we are just calling two functions that are already globally exposed I don't think we need a new function. We can just call those directly in the .talon file. That makes it more explicit.